aptos-move/framework/aptos-framework/sources/coin.spec.move

```diff
@@ -51,27 +51,27 @@ spec aptos_framework::coin {
         pragma verify = false;
     }
 
-    spec schema ExistCoinInfo<CoinType> {
+    spec schema AbortsIfNotExistCoinInfo<CoinType> {
         let addr = type_info::type_of<CoinType>().account_address;
         aborts_if !exists<CoinInfo<CoinType>>(addr);
     }
 
     spec name<CoinType>(): string::String {
-        include ExistCoinInfo<CoinType>;
+        include AbortsIfNotExistCoinInfo<CoinType>;
     }
 
     spec symbol<CoinType>(): string::String {
-        include ExistCoinInfo<CoinType>;
+        include AbortsIfNotExistCoinInfo<CoinType>;
     }
 
     spec decimals<CoinType>(): u8 {
-        include ExistCoinInfo<CoinType>;
+        include AbortsIfNotExistCoinInfo<CoinType>;
     }
 
     spec supply<CoinType>(): Option<u128> {
         // TODO: complex aborts conditions.
```

Comment:
Can you remind me what this TODO is for?

Reply:
It was marked as TODO because the aggregator modules and type_info made the spec complex. However, it should be specifiable now. I will tackle it in the next PR.